sweep/fee_bumper.go

```diff
@@ -1678,6 +1678,16 @@ func prepareSweepTx(inputs []input.Input, changePkScript lnwallet.AddrWithKey,
 		return 0, noChange, noLocktime, err
 	}
 
+	// We also need to account for the weight of the extra change output.
+	//
+	// NOTE: The weight estimation will not be quite accurate because the
+	// witness data is greater when overlay channels are used. But that
+	// that shouldn't be a problem since we will increase the fee rate
+	// incrementally.
+	extraChangeOut.WhenSome(func(o SweepOutput) {
```

Comment:
Awesome catch :rocket::fire:!! Should we add some test coverage for this as well?

Reply:
I think instead we should pass in the extra pk script to `getWeightEstimate`. The function was modified to accept multiple pk scripts for this reason: https://github.com/lightningnetwork/lnd/commit/eaea11e48fdbd63adb00a01fa7319822ddbf8ed3

Reply:
added a unit test.

Reply:
> yes it was addressed, now it is using the getWeightEstimate by adding the extraOutput to the ChangeScript slice.

Ah, I see that now. Thanks :pray:!